### DISCUSSION TITLE
Points System in GSSOC

### DISCUSSION THREAD

**Discussion by MrCodYrohit** [sanjay-kv/Open-source-Practice#828] [Q&A] [answered]
If I raised a PR(Level 1) and its get merged, then will i receive 10 points(for lvl 1) or 11 points(for lvl 1 + 1 points for PR).

Also If raise PR, and its doesn't get merged, then will i received 1 points as well?

Please if possible then share a link of guidelines where its mentioned explicitly.

**Comment by sanjay-kv** [comment, marked as answer]
11 points(for lvl 1 + 1 points for PR).
PR has to get merged for point 
https://www.loom.com/share/b08be9b4cde748f28b3776ed24248346?sid=b750b5c4-6e5e-4a79-addf-028be72027fc
